components/tabs/Tabs.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC, useState, ReactElement } from "react";
+import Tab from "./Tab";
+import { TabList } from "./Tabs.styled";
+import { TabProps } from "./Tab";
+interface Props {
+  children: Array<ReactElement<TabProps>>;
+}
+
+const Tabs: FC<Props> = ({ children }) => {
```

Comment:
would love to see this as a stateless version so that you could initilize this in a particular tab. not critical, but could see maybe a url linking to a particular tab